xarray/plot/__init__.py

```diff
@@ -3,6 +3,8 @@
 
 from .facetgrid import FacetGrid
 
+from .animate import animate_line
```

Comment:
I have a weak preference for `xarray.plot.animate.line` instead of `xarray.plot.animate_line`. 
Similarly, the method in `animate.py` could be `line` instead of `animate_line`

Reply:
@dcherian I tried to implement this but couldn't work out how to have the animation functions accessible simultaneously as any of
```python
xarray.plot.animate.line(darray, animate='time')
darray.plot(animate='time')
darray.plot.line(animate='time')
darray.plot.animate(animate='time')
darray.plot.animate.line(animate='time')
```

You can see in one of the new testcases that the last two of these syntaxes don't currently work, and I can't really work out why - there's something different between `_PlotMethods` and my `_AnimateMethods` which I'm not understanding.

Reply:
```
darray.plot.animate(animate='time')
darray.plot.animate.line(animate='time')
```
Do we really need these? They're equivalent to 
```
darray.plot(animate='time')
darray.plot.line(animate='time')
```
which is more concise and just as clear.

ping @shoyer for input.

Reply:
I agree that @dcherian's two suggestions (`darray.plot(animate='time')` and `darray.plot.line(animate='time')`) seem like enough for the user-level API.